tardis_kromer_plot.py

```diff
@@ -218,9 +224,9 @@ def line_mask(self):
     def lam_noint(self):
         """Wavelength of the non-interacting packets"""
         if self._lam_noint is None:
-            self._lam_noint = (
-                csts.c.cgs / (self.mdl.packet_nus[self.noint_mask])
-            ).to(units.AA)
+            self._lam_noint = (csts.c.cgs / (self.mdl.packet_nus[self.noint_mask])).to(
+                units.AA
+            )
         return self._lam_noint
```

Comment:
The default line length in black is 88 characters - you need to format with "-l 80" to force a line length of 80 characters